src/traits/i-access/i-access.ts

```diff
@@ -17,6 +17,7 @@ import SyncPromise from 'core/promise/sync';
 
 import type iBlock from 'super/i-block/i-block';
 import type { ModsDecl, ModEvent } from 'super/i-block/i-block';
+import { FOCUSABLE_SELECTOR } from 'traits/i-access/const';
```

Comment:
Напрашивается отбивка

```suggestion

import { FOCUSABLE_SELECTOR } from 'traits/i-access/const';
```